ccr/createCCRActor.php

```diff
@@ -141,7 +141,7 @@
     $e_InformationSystem = $ccr->createElement('InformationSystem');
     $e_Actor->appendChild($e_InformationSystem);
 
-    $e_Name = $ccr->createElement('Name', $row1['facility']);
+    $e_Name = $ccr->createElement('Name', htmlentities($row1['facility'] ?? ''));
```

Comment:
hi @stephenwaite , Just checking why the htmlentities() function call is being used here?

Reply:
The facility has an & in its name  :)

Reply:
Will text() function get what you need? Been avoiding htmlentities since I don't understand it :)